agent/xds/testdata/endpoints/transparent-proxy-with-peered-upstreams.latest.golden

```diff
@@ -47,7 +47,7 @@
     },
     {
       "@type": "type.googleapis.com/envoy.config.endpoint.v3.ClusterLoadAssignment",
-      "clusterName": "no-endpoints?peer=peer-a",
+      "clusterName": "no-endpoints.default.peer-a.external.1c053652-8512-4373-90cf-5a7f6263a994.consul",
```

Comment:
Endpoints with no PrimarySNI defaulted to envoyID. Now there is only one static format for cluster names.